fblldb.py

```diff
@@ -41,6 +41,7 @@ def loadCommandsInDirectory(commandsDirectory):
 def loadCommand(module, command, directory, filename, extension):
   func = makeRunCommand(command, os.path.join(directory, filename + extension))
   name = command.name()
+  helpText = command.description().split('\n', 1)[0] # first line of description
```

Comment:
You could also use `str.splitlines()` here instead, unless the `maxsplit=1` here is important to you.